.github/docker/Dockerfile.node

```diff
@@ -1,3 +1,3 @@
-FROM node:20.17.0-bookworm
+FROM node:22.9.0-bookworm
```

Comment:
This should become LTS shortly - went ahead and bumped the distroless image to match.